cmd/syncer/main.go

```diff
@@ -28,8 +28,9 @@ const (
 )
 
 var (
-	kubeconfig = flag.String("kubeconfig", "", "Config file for -from cluster")
-	clusterID  = flag.String("cluster", "", "ID of this cluster")
+	kubeconfig   = flag.String("from", "", "Kubeconfig file for -from cluster")
```

Comment:
Let's rename this variable to `fromKubeconfig` so there's never any notion of "the kubeconfig" in this context.

Reply:
Done